test/WebJobs.Script.Tests/Managment/VirtualFileSystemFacts.cs

```diff
@@ -19,9 +21,16 @@ namespace Microsoft.Azure.WebJobs.Script.Tests.Managment
 {
     public class VirtualFileSystemFacts : IDisposable
     {
-        private const string SiteRootPath = @"C:\DWASFiles\Sites\SiteName\VirtualDirectory0";
+        private const string HomePath = @"D:\home";
```

Comment:
Modifying these tests to be closer to App Service reality. In App Service on Windows, since HOME is always defined, [RootPath](https://github.com/Azure/azure-functions-host/blob/8cfc7fb6e5d814589eed914e1e690141778244f0/src/WebJobs.Script.WebHost/Management/VirtualFileSystem.cs#L51) will always be d:\home for the purposes of resolving vfs paths, which is why you have to specify the "site/wwwroot" portion on your requests. However, when HOME is not defined, on Windows we'll use ScriptPath which is the script root directory, which is what these tests were doing.